eip/EIPS/eip-4337.md

```diff
@@ -453,26 +453,26 @@ For security reasons, it is important that the generated contract address will d
 This way, even if someone can create a wallet at that address, he can't set different credentials to control it.
 The factory has to be staked if it accesses global storage - see [reputation, throttling and banning section](#reputation-scoring-and-throttlingbanning-for-global-entities) for details.
 
-NOTE: In order for the wallet to determine the "counterfactual" address of the wallet (prior its creation), 
+NOTE: In order for the wallet to determine the "counterfactual" address of the wallet (prior its creation),
 it should make a static call to the `entryPoint.getSenderAddress()`
 
 ### Entry point upgrading
 
 Accounts are encouraged to be DELEGATECALL forwarding contracts for gas efficiency and to allow account upgradability. The account code is expected to hard-code the entry point into their code for gas efficiency. If a new entry point is introduced, whether to add new functionality, improve gas efficiency, or fix a critical security bug, users can self-call to replace their account's code address with a new code address containing code that points to a new entry point. During an upgrade process, it's expected that two mempools will run in parallel.
 
 ## RPC methods
-
-### * eth_sendUserOperation
+### debug Namespace
```

Comment:
There are two "debug Namespace" titles.  I think this one should have been something else, as it precedes the "eth_" ones.

Reply:
Yes it should be eth namespace